index.html

```diff
@@ -2264,22 +2264,22 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
             </div>
           </li>
           <li>
-            <p data-lang="en">In principle, the size of Zhuyin annotations, including the phonetic symbols and the tone marks, should be 25% of the size of the base characters. In vertical Zhuyin, Zhuyin annotations should be placed center-aligned vertically to the base character while in horizontal Zhuyin, Zhuyin annotations should be placed center-aligned over the base character beneath. </p>
+            <p data-lang="en">In principle, the size of Zhuyin annotations, including the phonetic symbols and the tone marks, should be 25% of the size of the base characters. In vertical Zhuyin, Zhuyin annotations should vertically center-aligned to the base character while in horizontal Zhuyin, Zhuyin annotations should be center-aligned over the base character beneath it.</p>
```

Comment:
should vertically  ->  should be vertically

Reply:
maybe also a comma before 'while' ?

Reply:
'beneath it' doesn't agree grammatically with 'Zhuyin annotations' – maybe use 'them'

Reply:
Noted. Will make the changes.